react-components/src/components/RevealToolbar/Help/MenuSection.tsx

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { type ReactElement } from 'react';
+import { type HelpMenuSectionProps } from './types';
+import {
+  SectionContainer,
+  SectionTitle,
+  SectionSubTitle,
+  InstructionDetail,
+  SectionContent
+} from './elements';
+
+export const MenuSection = ({
```

Comment:
Nice, I think this is a good way of grouping the components :+1: Could we call this just `Section`? I think `Menu` is a bit misleading